accession-commons-mongodb/src/test/java/uk/ac/ebi/ampt2d/test/configuration/MongoDbTestConfiguration.java

```diff
@@ -80,7 +80,8 @@ public TestMongoDbInactiveAccessionService testMongoDbInactiveAccessionService()
     @Bean
     public AccessioningService<TestModel, String, String> testMongoDbAccessioningService() {
         return new BasicAccessioningService<>(
-                new SingleAccessionGenerator<>(o -> "id-" + o.getValue()),
+                new SingleAccessionGenerator<>(o ->
+                        "id-" + o.getValue()),
```

Comment:
Maybe you didn't intend this line to be split, at least I find it harder to read.